CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 - Added `WyciskOpenCircuitPotential` for differential capacity hysteresis state open-circuit potential submodel ([#3593](https://github.com/pybamm-team/PyBaMM/pull/3593))
 - Transport efficiency submodel has new options from the literature relating to different tortuosity factor models and also a new option called "tortuosity factor" for specifying the value or function directly as parameters ([#3437](https://github.com/pybamm-team/PyBaMM/pull/3437))
 - Heat of mixing source term can now be included into thermal models ([#2837](https://github.com/pybamm-team/PyBaMM/pull/2837))
+- Added a JAX interface to the IDAKLU solver ([#3658](https://github.com/pybamm-team/PyBaMM/pull/3658))
```

Comment:
I'm not sure if this was a product of the merge, but the new CHANGELOG entries are not sorted (and similarly for other sections below). Could you please fix the order?

Reply:
I just moved them to the correct section, I did not go through and sort. Is sorting actually important?

Reply:
Yes, it would be helpful when finding PRs because we keep the CHANGELOG in descending order, plus it looks nicer

Reply:
Also, I think these new additions are the only unsorted ones